### ISSUE TITLE
[Synthetics] Set up monitors page does not mention docker image requirements for Heartbeat option

### ISSUE THREAD

**Issue by kyungeunni** [elastic/observability-docs#1999]
In order to use Browser monitors with Heartbeat, users need to use provided docker image as mentioned in Heartbeat [documentation](https://www.elastic.co/guide/en/beats/heartbeat/8.3/monitor-browser-options.html):
>Browser based monitors can only be run in our Heartbeat docker image, or via the elastic-agent-complete docker image. For more information, see [Synthetic monitoring using Docker](https://www.elastic.co/guide/en/observability/8.3/synthetics-quickstart.html).

In Synthetics documentation, However, it does not mention anything about these requirements in both place:
- https://www.elastic.co/guide/en/observability/current/synthetic-run-tests.html#synthetic-monitor-choose-heartbeat
- https://www.elastic.co/guide/en/observability/current/uptime-set-up.html#uptime-set-up-choose-heartbeat

This can be really confusing for users when running it with Heartbeat binary directly as per the documentation, and then get the error message mentioning this limitation.

cc @andrewvc

**Comment by colleenmcginnis** [comment]
@kyungeunni I'm not sure what you mean. In the [Set up monitors](https://www.elastic.co/guide/en/observability/current/uptime-set-up.html#uptime-set-up-instructions) guide, we instruct users to use the Heartbeat Docker image for all kinds of monitors when using Heartbeat or the `elastic-agent-complete` Docker image when using Elastic Agent. Is this incorrect? 

| Heartbeat | Elastic Agent |
|---|---|
| <img width="776" alt="Screen Shot 2022-07-19 at 3 40 53 PM" src="https://user-images.githubusercontent.com/10479155/179844807-b8cf67ee-7d59-4dd8-aa4e-a2a3d5760616.png"> | <img width="772" alt="Screen Shot 2022-07-19 at 3 40 59 PM" src="https://user-images.githubusercontent.com/10479155/179844820-af290e01-8b07-447a-807f-93a52fdbedb4.png"> |

**Comment by kyungeunni** [comment]
Hi Colleen, My experience was this:

As a user with a bit of experience with Elastic products, I wanted to try out the Browser monitors this time. So I go to Synthetics monitoring doc. I see there are two approaches that I can choose, Heartbeats and Elastic agent. So I read the  [Heartbeat section](https://www.elastic.co/guide/en/observability/current/uptime-set-up.html#uptime-set-up-choose-heartbeat) and decided that this is the option I'd like to try. Then I skipped the _Set up_ instruction below because I already have Heartbeat binary on my local machine and the doc didn't mention any restriction or prerequisites. So I configured the heartbeat.yml, and tried to run the browser monitor using the binary, then I encountered the error message saying you must use Docker image provided by Elastic. It was a surprise for me, so I thought it'd be helpful If the section mentioned that I need to use Heartbeat docker image to run the browser monitor.
<img width="751" alt="image" src="https://user-images.githubusercontent.com/16660866/179947111-6441722c-b486-4b7c-8770-583e6c6b183e.png">

Maybe my experience was a bit peculiar, so I understand if it doesn't need any further attention in your opinion.
